tests/ha/check_cluster_integrity.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check cluster integrity
+# Maintainer: Christian Lanig <clanig@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+use utils 'systemctl';
+
+sub run {
+    select_console 'root-console';
+    my $cluster_name = get_cluster_name;
+
+    # Remove iptable rules in node 1 when testing qnetd/qdevice in multicast
+    assert_script_run "iptables -F && iptables -X" if (is_node(1) && check_var('QDEVICE_TEST_ROLE', 'client') && !get_var('HA_UNICAST'));
+
+    # Workaround network timeout issue during upgrade
+    if (get_var('HDDVERSION')) {
+        assert_script_run 'journalctl -b --no-pager -o short-precise > bsc1129385-check-journal.log';
```

Comment:
Could be easier to add the bsc# number in a variable, as it is write lot of time. But again nothing to worry about.